SwiftyBluetooth/Source/Central.swift

```diff
@@ -62,6 +62,18 @@ public enum PeripheralScanResult {
 }
 
 /**
+ The different results returned in the closure of the Central connect(...) and disconnect(...) functions.
```

Comment:
@tehjord Pulled this changes into our fork: https://github.com/SomaticLabs/SwiftyBluetooth/tree/result

Any idea when you plan to merge them into master?